.github/workflows/translation.yml

```diff
@@ -0,0 +1,275 @@
+            --overwrite true \
+            --account-name ${{ secrets.STORAGE_ACCOUNT_NAME }} \
+            --container-name $INPUT_CONTAINER_NAME \
+            --name ${{ vars.DOCUMENT_PATH }} \
+            --file ${{ vars.DOCUMENT_PATH }} \
+            --sas-token $sas
+  ###########################################
+  # Translate the document to all languages #
+  ###########################################
+  translate-language:
+    needs: upload-source
+    name: translate to ${{ matrix.language }}
+    env:
+      OUTPUT_CONTAINER_NAME: translatetarget${{ matrix.language-id }}
+    strategy:
+      matrix:
+        include:
+          - language: "Hebrew"
+            language-id: "he"
+            glossary: translation-glossary-he.csv
+          - language: "Arabic"
+            language-id: "ar"
+    runs-on: ubuntu-latest
+    steps:
+      ##########################
+      # Checkout the code base #
+      ##########################
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      ##########################
+      # Login to Azure         #
+      ##########################
+      - name: "Az CLI login"
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      ############################
+      # Create output container  #
+      ############################
+      - name: "Create containers with Azure CLI"
+        run: |
+          set -eux
+          az storage container create \
+            --name $OUTPUT_CONTAINER_NAME \
+            --account-name ${{ secrets.STORAGE_ACCOUNT_NAME }} \
+            --auth-mode login
+      #################################
+      # Upload translation, if exists #
+      #################################
+      - name: "Upload Translation"
+        if: matrix.glossary
+        run: |
+          set -eux
+
+          end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+          sas=`az storage container generate-sas -n $GLOSSARY_CONTAINER_NAME --auth-mode login --as-user --account-name ${{ secrets.STORAGE_ACCOUNT_NAME }} --https-only --permissions dlrw --expiry $end -o tsv`
+          az storage blob upload \
+            --overwrite true \
+            --account-name ${{ secrets.STORAGE_ACCOUNT_NAME }} \
+            --container-name $GLOSSARY_CONTAINER_NAME \
+            --name ${{ matrix.glossary }} \
+            --file translations/${{ matrix.language-id }}/${{ matrix.glossary }} \
+            --sas-token $sas
+          glossary_sas=`az storage container generate-sas -n $GLOSSARY_CONTAINER_NAME --auth-mode login --as-user --account-name ${{ secrets.STORAGE_ACCOUNT_NAME }} --https-only --permissions lre --expiry $end -o tsv`
+          glossary_uri_with_sas=https://${{ secrets.STORAGE_ACCOUNT_NAME }}.blob.core.windows.net/$GLOSSARY_CONTAINER_NAME/${{ matrix.glossary }}?$glossary_sas
```

Comment:
can't be fetched from the storage blob upload command?

Reply:
no, it has sas